sf.Rproj

```diff
@@ -0,0 +1,21 @@
+Version: 1.0
+
+RestoreWorkspace: No
+SaveWorkspace: No
+AlwaysSaveHistory: Default
+
+EnableCodeIndexing: Yes
+UseSpacesForTab: Yes
+NumSpacesForTab: 2
```

Comment:
To be consistent with the rest of the code, these would be "No" and 4, respectively. Do you see a problem there?